stock_barcodes_advanced/wizard/stock_barcodes_read_picking.py

```diff
@@ -0,0 +1,16 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import models
+
+
+class WizStockBarcodesReadPicking(models.TransientModel):
+    _inherit = "wiz.stock.barcodes.read.picking"
+
+    def update_fields_after_process_stock(self, moves):
+        self.product_id = False
+        self.product_qty = 0
+        self.lot_id = False
+        for record in self.pending_move_ids:
+            record._compute_qty_done()
```

Comment:
This method is multi, isn't it?
```suggestion
        self.pending_move_ids._compute_qty_done()
```